### PR TITLE
Consumer org management

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Adding information about the consumer org management

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/manage_marketplace/consumer_experience/_index.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Anonymous consumers can freely explore the published products in the Marketplace
 
 A *Private marketplace* can be browsed by navigating to the configured Marketplace URL, for example: `http://acme.marketplace.axway.com`. Consumers are redirected to a login screen and ask to enter their Platform credentials.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/_index.md:12]: 'asked'

File: content/en/docs/manage_marketplace/consumer_experience/_index.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ Anonymous consumers can freely explore the published products in the Marketplace
...
 
+There are two ways of getting credentials for a marketplace:
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/_index.md:14]: There are two ways of signing up in the Marketplace:

File: content/en/docs/manage_marketplace/consumer_experience/_index.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ Anonymous consumers can freely explore the published products in the Marketplace
...
+
+* a provider invited you in his organization
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/_index.md:16]: a provider invited you into his/her organization

File: content/en/docs/manage_marketplace/consumer_experience/_index.md [hunk 4]
```diff
@@ -11,6 +11,11 @@ Anonymous consumers can freely explore the published products in the Marketplace
...
+* provider decided to let his consumer to self signup to the marketplace and manage their [organization, users and teams](/docs/manage_marketplace/consumer_experience/consumer_organization)
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/_index.md:17]: consumers can self register to the Marketplace  and manage their [organization, users and teams](/docs/manage_marketplace/consumer_experience/consumer_organization)

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+## Objectives
+
+As a consumer, learn how to create and manage a consumer organization from the Marketplace.
+
+## Concepts
+
+### Creating a consumer organization
+
+The first time a user sign up to a marketplace, he has to enter the following information in the displayed form to create his account:
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:18]: The first time a user signs up to a marketplace, she must enter the following information in the sign up form:

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+* First name - first name of the user
+* Last name - last name of the user
+* Email - email address that will be use later to login to the marketplace
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:22]: email address that will be used to login to the marketplace

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+* Password - the password associated to the email to enable the login process to be successful. A confirmation of the password is mandatory
+* Organization - the name of the organization that will be created and reference as a consumer organization
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:24]: referenced

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+
+To submit the information, press the **Sign up** button. A message to Check your email is displayed. The email contains the link to activate your account. Once the activation is done, you will be able to login with your email and password into the marketplace.
+
+### Managing Users and Roles
+
+A consumer organization allows you to defines users and teams. Each user can have one or more role. Each can be part of one or multiple teams.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:30]: A user can have one or more role assigned, and can be a member of multiple teams.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+The first user who creates the organization is the **Administrator** of the organization. This administrator can change the roles of the users, with the restriction that there always needs to be at least one user that has the Administrator role.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:32]: The user that registered with a consumer organization will become the **Administrator** of the organization. The administrator can manage the user roles, under the condition that there always must be a user that has the administrator role assigned.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+
+This Administrator has all the roles on the Default team.
+
+We can distinguish the following types of roles:
+
+* *Organization Roles* - a role that applies to all the capabilities of the organization and is mutually exclusive. There are two Organization roles:
+    * **Administrator** - allow to create users and teams
+    * **Consumer** - allow to consumer product in marketplace
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:40]: consume products

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+* *Team Roles* - roles that are specific to a team the user belongs to. These roles are not mutually exclusive. A member can have multiple role in the the same team.  There are three team roles:
+    * **Consumer** - allow to manage free plan subscription, product access and credentials and application
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:42]: and to manage their applications

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+    * **Team Manager** - allow to organize the team: add/remove existing user
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:43]: allow to manage the team by adding or removing users

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+    * **Subscriber** - allows to manage the paid plan subscription
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:44]: allows to subscribe to paid plans and manage paid subscriptions.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+
+### Organizing the user within teams
+
+A team is a logical grouping of users. The idea is to enable you to create teams so that certain groups of people can work together or have access to the same set of Marketplace product. The same user can belong to multiple teams and can also have a different role in each team. Each organization always has a default team.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:48]: remove 'each organization always has a default team'

Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:48]: A team is a logical grouping of users that allows certain group of members to collaborate on the consumption of a product and building an application, but accessing same set of subscriptions, credentials and applications. A user can belong to multiple teams and have different roles assigned in each team.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+
+Only Administrator and Team Manager can create/view/edit/delete any team. But, the default team cannot be deleted. There should always be a Default team in an organization.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:50]: Each organization must have a default team.  Default team cannot be deleted.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+
+## User Management
+
+The user management is performed by an Administrator user.
+
+### Inviting a user to an organization
+
+For Adding a user to the current organization, Administrator has to invite the user.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:58]: A new user can be added to the organization by sending an invite.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+
+1. Navigate to Organization > Users
+2. click the **Add user** button
+3. A new screen is displayed where Administrator has to enter the user email and provide him the appropriate roles (Organization role and Team roles).
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:62]: Administrator must enter the user email and assign the appropriate Organization and Team roles.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+4. Click Save when done and the user will receive the invite email.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:63]: the email invite.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+3. A confirmation popup is displayed for the user to validate the action
+4. Once the action is validated, the user is removed from the system
+
+Administrator user cannot be removed if no other Administrator is present in the system
+
+## Team management
+
+Adding, editing, set default team and removing a team is only available for Administrator user.
+
+Team Administrator can only add/remove members from the team they are part of.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:95]: A Team Administrator can only add or remove members in the team they belong to.

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+1. Navigate to Organization > Teams
+2. Use the ellipsis menu **Edit**
+3. Edit the team name and/or description. Modify any user assignment to the team
+4. Click Save when done
+
+### Changing the Default Team
+
+1. Navigate to Organization > Teams
+2. Use the ellipsis menu **Set as Default**
+3. A confirmation popup is displayed for the user to validate the action to be sure he wants to replace the existing default team
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:115]: A confirmation popup is displayed for the user to validate the action

File: content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+4. Once the action is validated, the current team is now set a the Default team
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/consumer_organization.md:116]: as the Default team

File: content/en/docs/manage_marketplace/marketplace_consumer_organization.md [hunk 22]
```diff
@@ -0,0 +1,39 @@
+---
+title: Marketplace Consumer Organization
+linkTitle: Marketplace Consumer Organization
+weight: 40
+---
+
+Provider can provide one or more Consumer organization that are independently managed from his own organization
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_consumer_organization.md:7]: Providers can configure one or more Consumer organizations that are independently managed from their own provider organization.

File: content/en/docs/manage_marketplace/marketplace_consumer_organization.md [hunk 23]
```diff
@@ -0,0 +1,39 @@
...
+
+Understand what is a Consumer organization and learn how to enable/disable a Marketplace Consumer organization.
+
+## What is a Consumer organization?
+
+When you want to share your marketplace with multiple external entities (partners / subsidiaries ...) to your company, it becomes complex to manage all the users and associated teams especially when your Marketplace is connected to an IDP.
+
+To facilitate that, it is possible to delegate this user management using one or more Consumer organizations. A Consumer organization is a place where consumers will be able to self-organize their users and teams independently from your provider organization.
+
+For instance, if you have a partner and you don't want to manage their users in your provider organization, you can let them have their own Consumer organization. Like this your partner becomes self-autonomous to manage their users/teams and you don't need to bother to introduce their users into your user management.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_consumer_organization.md:23]: We should remove this entire paragraph.

File: content/en/docs/manage_marketplace/marketplace_consumer_organization.md [hunk 24]
```diff
@@ -0,0 +1,39 @@
...
+
+Refer to [Consumer organization management](/docs/manage_marketplace/consumer_experience/consumer_organization)
+
+## View the Marketplace consumer organization
+
+From platform.axway.con > Marketplace > navigate to **Consumer Orgs** page.
+
+This page displays all the Consumer organization available with their number of teams and users. Provider is not able to add teams or users to those organizations and can only enable/disable them.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_consumer_organization.md:31]: A provider does not have the rights to add teams or users into a consumer organization. The only actions they are allowed to perform is Enable/Disable organizations.

File: content/en/docs/manage_marketplace/marketplace_consumer_organization.md [hunk 25]
```diff
@@ -0,0 +1,39 @@
...
+
+Only Enabled organization allows user to connect to the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_consumer_organization.md:33]: Disabling a Consumer Organization will stop users registered in that organization to connect to the Marketplace.

File: content/en/docs/manage_marketplace/marketplace_consumer_organization.md [hunk 26]
```diff
@@ -0,0 +1,39 @@
...
+
+## Enable and disable a Consumer organization
+
+Select an active organization you want to disable and clicks the ellipsis menu / Disable. A confirmation popup is displayed. Clicking the **Disable** button will trigger the action and the organization will be disabled. **Cancel** button will cancel the action and the organization will stay in enable state. In a disabled state no user from this organization will be able to connect to the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_consumer_organization.md:37]: click

File: content/en/docs/manage_marketplace/marketplace_settings.md [hunk 27]
```diff
@@ -32,6 +32,11 @@ Learn how to use the Amplify Enterprise Marketplace WebUI to configure your orga
         * **Protected** - select to make the Marketplace accessible only to registered users in your organization.
         * **Public** - select to make your Marketplace accessible to all users, anonymous and authenticated with an account.
 
+    * **Self signup** - indicates whether a user can signup for the Marketplace:
+
+        * **Disallow** - nobody can signup to the Marketplace and only Administrator can invite users to ths Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_settings.md:37]: **Disallow** - self-signup is disabled  and only Administrators can invite users into the Marketplace.

File: content/en/docs/manage_marketplace/marketplace_settings.md [hunk 28]
```diff
@@ -32,6 +32,11 @@ Learn how to use the Amplify Enterprise Marketplace WebUI to configure your orga
...
+        * **Allow** - anyone can signup to the Marketplace and register their own account. They will receive and email to activate their account.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/marketplace_settings.md:38]: remove *They will receive and email to activate their account* as it is not relevant here

**Review by georgiana-caras — CHANGES_REQUESTED** [review]
please make the highlighted changes before publishing the documentation (Contains rephrasing of content, correction of typos)

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**